### PR TITLE
fix(2312): add latest to template and command

### PR DESCRIPTION
**Pull request by klu909**
## Context

adding a latest field to template/command model
## Objective

update latest field when creating new template/command

## References

https://github.com/screwdriver-cd/screwdriver/issues/2312

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: lib/commandFactory.js [hunk 1]
```diff
@@ -74,9 +74,13 @@ class CommandFactory extends BaseFactory {
                 const patch = parseInt(latestPatch.slice(1), 10) + 1;
 
                 config.version = `${latestMajor}${latestMinor}.${patch}`;
+                latest.latestVersion = false;
+
+                await latest.update();
```
Comment (tkyi) [inline lib/commandFactory.js]: does the function on line 64 need `async` since you're using `await` here?

File: lib/templateFactory.js [hunk 2]
```diff
@@ -156,9 +156,13 @@ class TemplateFactory extends BaseFactory {
                 const patch = parseInt(latestPatch.slice(1), 10) + 1;
 
                 result.version = `${latestMajor}${latestMinor}.${patch}`;
+                latest.latestVersion = false;
+
+                await latest.update();
```
Comment (tkyi) [inline lib/templateFactory.js]: same comment as above

File: lib/commandFactory.js (hunk 1 above)
Comment (jithine) [reply lib/commandFactory.js]: yep, hence the build failure

**Review by tkyi — COMMENTED** [review]
any way to test this change?

**Comment by klu909** [comment]
> any way to test this change?

not really..

**Review by tkyi — APPROVED** [review]

**Pull request merged**